content/blogposts/2022/code-intelligence-platform.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "From code search toward a code intelligence platform"
+publishDate: 
+description: TODO
+tags: [blog]
+slug: "code-search-to-code-intelligence"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+> NOTE: This post is very WIP. To be edited and tightened up a lot.
+
+No matter how good your other dev tools, coding is still complex. Even if you've configured your editor just right, sped up your test suite, set up continuous deployment, added observability, etc., coding is still complex. Higher-level languages, libraries, frameworks, cloud providers, etc., that encapsulate complexity just let you spend your complexity budget elsewhere. The overall complexity of code has increased, not declined, over time, and there's no sign of that trend changing.
```

Comment:
@sqs - Is there a way to say dev success = business success without coming across as too calculated? In the Google doc, @amieroth said  "I like what you talk about with the growing importance of devs within a company and how that's evolved in recent years." I think that line of messaging works well (and more implicitly states dev success = business success).

Reply:
Added 2 sections below